### DISCUSSION TITLE
what tool is used to create diagrams in nextjs documentation website?

### DISCUSSION THREAD

**Discussion by medinLaVie** [vercel/next.js#63957] [Help]
### Summary

Anyone know what's the tool used to create nextjs diagram? 

I think those diagrams look really cool, and I want to learn how to create them. Thanks!

[](url)
<img width="661" alt="Screenshot 2024-04-01 at 22 08 59" src="https://github.com/vercel/next.js/assets/3932566/2f8bb040-e1ce-4d13-8a1d-c444aa4f9928">

**Comment by truexor** [comment]
Probably used `Photoshop`, `Illustrator` or some other photo editing software.
If keeping ourselves solely to web UI category itself, `Figma` is the go-to platform.
